edx_proctoring/tests/test_api.py

```diff
@@ -1451,6 +1452,30 @@ def test_get_studentview_started_timed_exam(self):
         )
         self.assertIsNone(rendered_response)
 
+    @ddt.data(True, False)
+    def test_get_studentview_long_limit(self, under_20_hours):
+        """
+        Test for hide_extra_time_footer on exams with > 20 hours time limit
+        """
+        exam_id = self._create_exam_with_due_time(is_proctored=False, )
+        if under_20_hours:
+            update_exam(exam_id, time_limit_mins=((20 * 60) - 1))  # 1 minute less than 20 hours
+        else:
+            update_exam(exam_id, time_limit_mins=((20 * 60) + 1))  # 1 minute greater than 20 hours
```

Comment:
Might be nice to explicitly test what happens if it equals 20 hours, or to note that what happens for that case is currently the same as the 1 minute less than 20 hours case.